marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/fastfunctest/TestSearchPageLength.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2024 MarkLogic Corporation. All Rights Reserved.
+ */
+
+package com.marklogic.client.fastfunctest;
+
+import com.marklogic.client.impl.OkHttpServices;
+import com.marklogic.client.impl.RESTServices;
+import com.marklogic.client.io.*;
+import com.marklogic.client.query.*;
+import org.custommonkey.xmlunit.exceptions.XpathException;
+import org.junit.jupiter.api.*;
+import org.w3c.dom.Document;
+
+import static org.custommonkey.xmlunit.XMLAssert.*;
+
+
+class TestSearchPageLength extends AbstractFunctionalTest {
```

Comment:
I have no idea why so many tests start with "Test" instead of ending with "Test", when the latter is the very common convention for JUnit. Having everything start with "Test" is also annoying for finding test classes via meaningful verbs at the start of the test class. So do `SearchWithPageLengthTest` to make it obvious what this test class is covering. 

Also, you can see that not much thought was given to packaging structures - i.e. there's a ton of tests in one package. We can start improving that though - so do `com.marklogic.client.fastfunctest.search` for the package.